client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -104,6 +116,28 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 					} )
 				);
 
+			case 'PAYMENT_METHOD_SELECT': {
+				reduxDispatch( logStashEventAction( 'payment_method_select', String( action.payload ) ) );
```

Comment:
For probably poor reasons, I made `logStashEventAction` always have the logstash message `composite checkout load error`, which we probably don't want here. It may be worth modifying that function to have an optional message body.